intl/messages/fr.json

```diff
@@ -0,0 +1,135 @@
+    "list": [
+      {
+        "title": "Exécuter sur Node.js et le navigateur",
+        "desc": "js-ipfs ne requiert aucune configuration et fonctionne sur Node.js, Electron et n'importe quel navigateur moderne. Il est également personnalisable par conception afin que vous puissiez l'utiliser avec votre ‘runtime’ préféré."
+      },
+      {
+        "title": "Implémentation IPFS complète",
+        "desc": "Aucune fonctionnalité n'a été oublié. js-ipfs n'est pas un client léger, c'est l'implémentation complète du protocole IPFS."
+      },
+      {
+        "title": "Utiliser PubSub pour communiquer en temps réel avec les autres pairs",
+        "desc": "Les nœuds IPFS peuvent créer des topologies réseau basées sur des sujets d'intérêt pour diffuser des événements en temps réel."
+      },
+      {
+        "title": "Ajouter et récupérer des fichiers de n'importe où dans le réseau IPFS",
+        "desc": "IPFS est conçu pour utiliser la puissance de l'adressage de contenu et de l'adressage de processus pour trouver les nœuds du réseau ayant le contenu que vous recherchez et les récupérer. De la même manière, vous pouvez ajouter n'importe quel élément de données et d'autres nœuds les trouveront aussi."
+      },
+      {
+        "title": "Utilise l'API DAG pour traverser toute les hash lié de structure de données",
+        "desc": "IPFS utilise [IPLD] (https://ipld.io/), l'InterPlanetary Linked-Data, un modèle qui vous permet d'interagir avec des données provenant de plusieurs sources, telles que git, blockchains et plus encore."
+      },
+      {
+        "title": "libp2p est intégré",
+        "desc": "libp2p est le ‘stack’ de réseau modulaire qui a été créée pour IPFS et maintenant vous pouvez l'utiliser directement via IPFS pour votre application."
+      },
+      {
+        "title": "Exécuter en tant que daemon",
+        "desc": "js-ipfs a aussi l'option de l'exécuter en tant que daemon pour que vous puissiez interagir avec lui en utilisant l'API HTTP que vous connaissez depuis go-ipfs."
+      },
+      {
+        "title": "Créer des adresses stables pour changer les données",
+        "desc": "Il est livré avec IPNS, le système de dénomination interplanétaire, un moyen d'écrire des pointeurs mutables (enregistrements) pour distribuer les mises à jour de manière authentifiée et certifiée."
+      }
+    ]
+  },
+  "gettingStarted": {
+    "sectionTitle": "Commencer",
+    "sectionDesc": "IPFS gives you a Content Addressing primitive for all your data on the DWeb. You can make data available to the network or fetch existing data through its CID, the Content Identified. Try it out below!",
+    "addDataToIPFS": "Adding data to IPFS",
+    "output": "Output",
+    "getDataFromIPFS": "Getting data from IPFS",
+    "usingJavascript": "Using Javascript in the Browser or Node.js",
+    "usingCli": "Using the CLI",
+    "usingGateway": "Using the HTTP Gateway"
+  },
```

Comment:
@camillecoyac, the Getting Started got an update meanwhile. Do you mean translating these sentences? Apologies for the extra work!